### PR TITLE
Update AboutFooter snapshot and mock Date function

### LINKED ISSUE #2639: Happy New Year!
We're failing CI with a front-end snapshot test that is hard-coding the year 2021.  Welcome to 2022 everyone!

See:

- https://github.com/Seneca-CDOT/telescope/runs/4693402544?check_suite_focus=true
- https://github.com/Seneca-CDOT/telescope/blob/d751549c04d99d366f88bed624cdda5ff2b40fa5/src/web/src/components/__snapshots__/AboutFooter.test.tsx.snap#L155

### PR DESCRIPTION
**Pull request by DukeManh**
## Issue This PR Addresses
Closes #2639 

This PR updates the year to 2022 in `AboutFooter` snapshot and mocks the Date function so we don't repeat the issue next year.

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/web/src/components/AboutFooter.test.tsx [hunk 1]
```diff
@@ -2,6 +2,12 @@ import { render, cleanup } from '@testing-library/react';
 
 afterEach(cleanup);
 
+// Make sure we get a consistent date regardless of when the test is run
+const milliseconds = 1641234611041; // 2022-01-03T18:30:11.041Z
```
Comment (humphd) [inline src/web/src/components/AboutFooter.test.tsx:6]:
```js
const milliseconds = new Date('2022-01-03T18:30:11.041Z').getTime();
```

**Review by humphd — CHANGES_REQUESTED** [review]
Excellent, one small fix.

File: src/web/src/components/AboutFooter.test.tsx (hunk 1 above)
Comment (DukeManh) [reply src/web/src/components/AboutFooter.test.tsx:6]: Great.

File: src/web/src/components/AboutFooter.test.tsx [hunk 2]
```diff
@@ -2,6 +2,11 @@ import { render, cleanup } from '@testing-library/react';
 
 afterEach(cleanup);
 
+// Make sure we get a consistent date regardless of when the test is run
+const mockDate = new Date('2022-01-03T18:30:11.041Z');
+jest.spyOn(global, 'Date').mockImplementation(() => mockDate as any);
+Date.now = jest.fn(() => mockDate.getTime());
```
Comment (humphd) [inline src/web/src/components/AboutFooter.test.tsx:8]: This is even better, eliminating the need for `milliseconds` altogether.  Nice improvement.

**Review by humphd — APPROVED** [review]

**Comment by humphd** [comment]
Sorry to merge with only 1 review, I don't want to leave CI breaking any longer than necessary.

**Pull request merged**